### DISCUSSION TITLE
Keyboard shortcuts do not work in Cyrillic layout

### DISCUSSION THREAD

**Discussion by waitstop** [zed-industries/zed#20135] [Feature Requests]
I have a problem that I cannot use any keyboard shortcuts in Russian or Ukrainian layout, even `ctrl+c`, `ctrl+w` doesn't work. Everything works on the Czech and English layouts.
If it's important, I use: arch with hyprland. 
My config for input:
```
input {
    kb_layout = us,ru
    kb_options = grp:win_space_toggle
    follow_mouse = 1
}
```
Is there something wrong with zed or is it my system settings?

**Comment by SomeoneToIgnore** [comment]
With Zed: https://github.com/zed-industries/zed/issues/14053
